bin/cli.js

```diff
@@ -142,6 +145,12 @@ function syncShorthand (opts) {
   return usage(opts)
 }
 
+// This was needed so that we can show help messages from extensions
+function showUsageOrRunExtension (opts, help, usageMessage) {
```

Comment:
technically it still runs via the other path normally, this is only used when the `--help` flag is passed.

Reply:
Any suggestions on how I should word it instead? 😀

Reply:
`showUsageOrRunExtentionToShowUsage` ?

its fine =)

Reply:
I'll just leave it as is then. :P